app/views/play/modal/PollModal.coffee

```diff
@@ -88,7 +88,13 @@ module.exports = class PollModal extends ModalView
       btn = @$el.find('.btn.btn-illustrated.btn-lg.done-button')
       btn.off('click')
 
-      if nextPollId
+      if answerObj.nextURL
+        btn.text(i18n.t('common.next'))
+        btn.one('click', () =>
+          btn.prop('disabled', true);
+          window.open(answerObj.nextURL, '_blank')
```

Comment:
if its external url, security review had asked us to add: `rel="noopener noreferrer"`

Reply:
good point. i can add this